transfers/transfer.py

```diff
@@ -16,6 +16,10 @@
 import subprocess
 import sys
 import time
+import ConfigParser
```

Comment:
Python3 renamed this to [configparser](https://docs.python.org/3/library/configparser.html) Maybe we should add six as a dependency at this point? Or
```
try:
    import ConfigParser as configparser
except ImportError:
    import configparser
```